tests/test_filters.py

```diff
@@ -118,6 +118,12 @@ def test_filters_game(self):
         self.message.game = None
         self.assertFalse(Filters.game(self.message))
 
+    def test_filters_successful_payment(self):
+        self.message.successful_payment = 'test'
+        self.assertTrue(Filters.successful_payment(self.message))
+        self.message.successful_payment = None
+        self.assertFalse(Filters.successful_payment(self.message))
+
```

Comment:
Could you add a test for invoice too? I know you didn't actually add the filter, but i just saw that it was missing.